ui/src/app/views/app/component.spec.ts

```diff
@@ -1,11 +1,16 @@
 import {TestBed, async} from '@angular/core/testing';
 import {Title} from '@angular/platform-browser';
-import {AppComponent} from 'app/views/app/component';
 import {RouterTestingModule} from '@angular/router/testing';
+
 import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {ClarityModule} from 'clarity-angular';
+
+import {AppComponent} from 'app/views/app/component';
 import {SignInService} from 'app/services/sign-in.service';
+import {ProfileService} from 'generated';
```

Comment:
Did we want generated classes separate from our hand written classes?

Reply:
Fixed. (I think the answer is "yes" from our style discussion, right?)